### PR TITLE
Implemented PoyntingVector module

### PR DESCRIPTION
**Pull request by ojha-aditya**
Hereby I have written the functions to implement the Poynting vector for plane polarized electromagnetic waves. The PR contains the following in the PoyntingVectorModule folder:

1. Module PoyntingVector.py which implements the functions to generate the electromagnetic wave expressions and Poynting vector expressions and calculate the numerical value for the same.
2. TestPoyntingVector.py which is the unit test module for PoyntingVector.py
3. Markdown file contains the Physics of Poynting vector calculations done using the above module.
4. There is a separate Jupyter notebook that can be used the test the functions individually.

### CODE REVIEW

**Comment by laserlab** [comment]
@Tolani4 Please review asap

File: PoyntingVectorModule/PoyntingVector.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    Snum = poynting_vector_value(E, B, [x, y, z], time)
+    Smag = poynting_vector_magnitude(Snum)
+
+    # Outputs all results
+    print("Electric Field (in V/m): ", E)
+    print("Magnetic Field (in T): ", B)
+    print("Poynting Vector (in W/m^2): ", S, " = ", Snum)
+    print("The magnitude of Poynting Vector (in W/m^2): ", Smag)
+
+if __name__ == "__main__":
```
[the remark this answers is not in the document]
Comment (ojha-aditya) [reply PoyntingVectorModule/PoyntingVector.py:87]: So I should remove the main() function loop basically from the module?

There is no column.py in my PR though.

File: PoyntingVectorModule/PoyntingVector.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    nz = float(input("Please enter z-component of polarisation: "))
+    x = float(input("enter the x-coordinate of position (in m): "))
+    y = float(input("enter the y-coordinate of position (in m): "))
+    z = float(input("enter the z-coordinate of position (in m): "))
+    time = float(input("Please enter time (in sec): "))
+
+    # calculating the Electric, Magnetic Field and Poynting vector
+    E = electric_field_expression(E_0, [kx, ky, kz], delta, f, [nx, ny, nz]) 
+    B = magnetic_field_expression(E_0, [kx, ky, kz], delta, f, [nx, ny, nz])
+    S = poynting_vector_expression(E, B)
```
[the remark this answers is not in the document]
Comment (ojha-aditya) [reply PoyntingVectorModule/PoyntingVector.py:77]: How do I modify it then? I am sorry I didn't quite understand

**Comment by ojha-aditya** [comment]
Also, should I just remove the Jupyter notebook from this PR as it is not longer needed? I had kept it just for trial run purposes.

**Comment by laserlab** [comment]
> Also, should I just remove the Jupyter notebook from this PR as it is not longer needed? I had kept it just for trial run purposes.

sure remove it if it is not needed

**Comment by ojha-aditya** [comment]
Done. Removed the Jupyter notebook, will soon commit. Also, is there a way to reflect the lint test results in our codespaces so it could be compared and fixed line by line?

**Comment by ojha-aditya** [comment]
@laserlab I have comitted all changes requested and got the linting score up as well. Removed the main function and loop, the remaining linting suggestions are a tradeoff it seems, resolving them causes another issue to pop up.

**Comment by laserlab** [comment]
merging with wrong formatted docstrings and linting issues

**Pull request merged**